### ISSUE TITLE
Accessibility internal audit of the application

### ISSUE THREAD

**Issue by adamjarling** [avalonmediasystem/structured-metadata-editor-react#64]
### Description
We should determine the application's current state of accessibility, make sure there are no red flags, and identify improvement areas.

### Done Looks Like
- Audit performed, and a summary of the application's current state is known and documented.

### To Do
- [x] Install WAVE browser extension tool, and become familiar with it.   https://wave.webaim.org/extension/
- [x] Analyze and document WAVE results in this issue?

**Comment by Dananji** [comment]
@adamjarling This looks good. Shall I pull this in? And feel free to change the size/estimate of this as you see fit :)

**Comment by adamjarling** [comment]
Yea, seems like a good place to put effort...maybe run it by @joncameron or @mcwhitaker ?

**Comment by Dananji** [comment]
@adamjarling I went through the WCAG 2.1 standard for evaluating the accessibility of the application. I used WAVE extension, Accessibility audit tool in Dev tools in Chrome, and Chrome VOX (screen reader) extension for Chrome to evaluate the success criteria of each sub section of the 13 guidelines in WCAG 2.1 to check whether the application conform with Level AA standard.
And I prepared the attached document based on the results of the audit. If this is not what you had in mind with respect to documenting the accessibility issues, please let me know what would be the ideal outcome of this evaluation (this is the first time I have done an accessibility audit, so please let me know if anything needs to change :) )
[Structural Metadata Editor - Accessibility Audit.pdf](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c0e783fd9fd29001136b2d2/125516c66e82c728ace21e0d46db8088368197bc9de8b3038958e4952d4733726921a722648af168f2025a020c0a0def180a0618a5bdc03bd7f36829ce4e17ff912f002e936e55beb2901ab805ec21891c8ce2ef8c1ec1ea3d22e78f8a1e6819051e9be34f77f1c71f9705af01b40f47ae01376aec77337944921f7c296c6f20a8fccd1b.pdf)

**Comment by adamjarling** [comment]
This work in the report is incredible, and extremely helpful.  @Dananji and I have been chatting about how to move on the findings of her audit.  She would like to include a "Recommended Actions" section of the report, and we'll both start creating Github issues from the `Fail` line items in the audit.  

For organization, this group of issues should be labeled by both the `structural-metadata-editor` and `accessibility` labels.

**Comment by Dananji** [comment]
Edited audit report with the `Recommended Actions` section based on the findings;
[Structural Metadata Editor - Accessibility Audit.pdf](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c0e783fd9fd29001136b2d2/125516c66e82c728ace21e0d46db8088368197bc9de8b3038958e4952d4733726921a722648af168f2025a020c0a0def180a0618a5bdc03bd7f36829ce4e17ff912f002e936e55beb2901ab805ec21891c8ce2ef8c1ec1ea3d22e78f8a1e6819051e9be34f77f1c71f9705af01b40f47ae013764ec773379449317752f696e25acfdcd1b.pdf)
